packages/playground/config/polyfills.js

```diff
@@ -19,10 +19,3 @@ import 'whatwg-fetch';
         });
     });
 })([Element.prototype, CharacterData.prototype, DocumentType.prototype]);
```

Comment:
Will the new polyfills remove the need for this `ChildNode.remove()` one?

Reply:
(same for `fetch`)

Reply:
If `fetch` is polyfiled by `core-js-pure`, we might be able to remove the dependency `whatwg-fetch`

Reply:
If `fetch` is polyfiled by `core-js-pure`, we might be able to remove the dependency `whatwg-fetch`

Reply:
I just left this for the playground because there is still code we are not transpilling in there.

Reply:
Sorry, I wasn't clear enough. I didn't mean in the playground; we use `whatg-fetch` as polyfill in our `lib` , on `src/core/Services/fetch.ts` . Can we ditch that polyfill too?

Reply:
we cannot remove this, using corejs there where errors on IE11